app/settings/users/filter-users.directive.js

```diff
@@ -24,6 +24,7 @@ function (
             _
         ) {
             $scope.filtersMenuOpen = false;
+            $scope.sortalph="realname";
```

Comment:
You can remove this now since we don't use it anymore.

Reply:
@Angamanga I have removed the unnecessary line of code